scripts/add_dummy_data_to_db.py

```diff
@@ -1,14 +1,26 @@
 # This script is useful if you want to test the dashboard with dummy data.
-# Run it using the following command:
-#    > PYTHON_PATH="../../../" python add_dummy_data_to_db.py
+# Navigate to the root directory of the project and run the following command:
+#   > python scripts/add_dummy_data_to_db.py
 
 import os
 import random
+import sys
 from datetime import datetime, timedelta
 
 import numpy as np
 from sqlalchemy.orm import Session
 
+# Append the framework path. NB: This is required if this script is invoked from the
+# command line. However, it is not necessary if it is imported from a pip install.
+if __name__ == "__main__":
+    PACKAGE_PATH = os.path.abspath(__file__)
```

Comment:
/nit I'd suggest used pathlib instead of os.path. https://docs.python.org/3/library/pathlib.html

Reply:
Way nicer to read.

Reply:
will make the change!